trame_client/utils/testing.py

```diff
@@ -11,19 +12,23 @@
 
 
 class TrameServerMonitor:
-    def __init__(self, log_path):
+    def __init__(self, log_path, do_print_log_lines=True):
```

Comment:
Having a default to True, prevent you to turn off

Reply:
I had left the default behavior to the existing one and planned to use it as follows : 

```
@pytest.fixture
def server(xprocess, server_path):
    name, Starter, Monitor = HELPER.get_xprocess_args(server_path)

    # ensure process is running and return its logfile
    logfile = xprocess.ensure(name, Starter)
    try:
        yield Monitor(logfile[1], do_print_log_lines=False)
    finally:
        # clean up whole process tree afterwards
        xprocess.getinfo(name).terminate()
```

I can set the default to False if that's ok with you.

Reply:
I mean calling it like that won't work as the True default value will superseed your input.

Reply:
=> you do need to set the default to False

Reply:
Thx, I just changed the flag to False by default.